best_practices/source_control/terra_source_control_cheatsheet.md

```diff
@@ -0,0 +1,256 @@
+
+     <img src="./images/jupyter_upload.png" width="50%">
+
+     *Need help with this step? See [Creating your SSH keys](./terra_source_control_II.md#create-your-ssh-keys-on-your-laptop-or-desktop-computer).*
+
+      > **Best practice**: your SSH key should require a passphrase.
+
+      > **Tip**: if you create a new SSH key, don't forget to [add it to GitHub](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)!
+
+  4. Create a terminal:
+
+        * If you are in the Jupyter console, click on the `Files` tab, pull down the `New` menu towards the upper right of the screen, and choose `terminal`.
+        * If you are on a workspace page, you can click on the terminal icon `>_` on the right hand side of the page.
+
+### (2) Use the terminal to set up and clone your Git repository
+
+Run the following commands from the Terminal.
+
+  1. Tell Git who you are.
+      ```sh
+      # Edit these commands to have your name and your institutional email address.
+      git config --global user.email "me@institution.org"
+      git config --global user.name "Firstname Lastname"
+      ```
+      > *Need help with this step? See section in in `terra_source_control_II.md` on [Git setup](./terra_source_control_II.md#git-setup).*
+
+  2. Configure `ssh-agent` to start every time you log in.
+
+        ```sh
+        echo 'eval "$(ssh-agent -s)"' >> $HOME/.bashrc
+        source $HOME/.bashrc
+        ```
+        > *Need help with this step? See the section in `terra_source_control_II.md` on [setting up an ssh-agent](./terra_source_control_II.md#set-up-your-ssh-key-on-your-cloud-environments-persistent-disk).*
+
+  3. Create a directory to hold your GitHub ssh key.
+
+      ```sh
+      mkdir -p ~/.ssh
+      ```
+
+  4. Add your ssh key.
+
+      ```sh
+      # Edit these commands to use the path to your ssh key.
+      cd ~/.ssh
+      mv ~/my-GitHub-ssh-key .  # Move your key file from whereever it was uploaded to this dedicated directory.
+      chmod go-rwx my-GitHub-ssh-key
+      ssh-add my-GitHub-ssh-key
+      cd  # change back to your home directory
+      ```
+      > *Need help with this step? See the section in `terra_source_control_II.md` on [setting up an ssh-agent](./terra_source_control_II.md#set-up-your-ssh-key-on-your-cloud-environments-persistent-disk).*
+
+  5. Clone the repository. If you like, you can create a sub-directory into which you clone all your
+     repos.  In this example, we're just cloning into the home directory.
+
+      Note: **this command is just an example**. If you need to commit code to a different
+      repository, be sure to pass the correct value to `git clone`.
+
+      ```sh
+      git clone git@github.com:all-of-us/workbench-analysis-dev-tools.git
```

Comment:
oops, good catch.  Let's go with the `ukb-cross-analysis-demo-project `one as I think it might be of more immediate interest to much of the intended audience.